cache-manager/cache-manager.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for cache-manager v1.2.0
+// Project: https://github.com/BryanDonovan/node-cache-manager
+// Definitions by: Simon Gausmann <https://github.com/GausSim/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+declare module 'cache-manager' {
+
+
+    interface ICachingConfig {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files
> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.